run/idp-sql/credentials.py

```diff
@@ -16,19 +16,14 @@
 import os
 from typing import Dict
 
-from google.cloud import secretmanager
-
 from middleware import logger
 
 
 # [START cloudrun_user_auth_secrets]
 def get_cred_config() -> Dict[str, str]:
     if "CLOUD_SQL_CREDENTIALS_SECRET" in os.environ:
```

Comment:
Probably easier to do:

```
secret = os.environ.get("CLOUD_SQL_CREDENTIALS_SECRET")
if secret: 
  return json.loads(secret)
```

If you use `os.environ.get(....)`, it will return the value if it exists, but will return None if it doesn't.   You can also make it default to some useful value.